docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
 participant "m:Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("delete 1")
+[-> LogicManager : execute("delete 1 2 3")
```

Comment:
Great update showing how logic manager supports batch deletion of commands by parsing of multiple indexes.